pkg/util/containerd/containerd_util.go

```diff
@@ -363,6 +364,7 @@ func (c *ContainerdUtil) Status(namespace string, ctn containerd.Container) (con
 // It checks the io.cri-containerd.kind label
 // Ref:
 // - https://github.com/containerd/cri/blob/release/1.4/pkg/server/helpers.go#L74
+// TODO: Delete since it's not used anymore.
```

Comment:
I'm going to do this in a separate PR since it requires changing the ContainerdItf interface https://github.com/DataDog/datadog-agent/blob/main/pkg/util/containerd/containerd_util.go#L70C3-L70C3